app/actors/ProcessingSupervisor.scala

```diff
@@ -52,9 +52,9 @@ class ProcessingSupervisor @Inject()(applicationConfig: ApplicationConfiguration
     override def tryLock[T](body: => Future[T])(implicit ec : ExecutionContext): Future[Option[T]] = {
       repo.lock(lockId, serverId, forceLockReleaseAfter)
         .flatMap { acquired =>
-          if (acquired) { body.map { case x => Some(x) } }
+          if (acquired) { body.map { case x => Some(x) }(global) }
```

Comment:
Was this global required to make it compile and run tests

Reply:
yes it was

Reply:
as was seeing ec and global, with the implicit imports so had to declare